sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -70,7 +70,7 @@ def _from_generated(cls, generated, **kwargs):
             digest=generated.digest,
             last_updated_on=generated.last_updated_on,
             operating_system=generated.operating_system,
-            size=generated.size,
+            size_in_bytes=generated.size,
```

Comment:
I didn't realize that the service was using `size` here.... that makes it less appealing that we made this name longer...

Reply:
But other languages are using the longer name.

Reply:
Yeah, it would be good for the sake of consistency. Unless there's strong objection to name it like this in Python.